pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "otter-welcome-buddy"
```

Comment:
I think *otter buddy* is better for being shorter, but if you prefer we can update that after deprecating the other repo.

Reply:
I will use this bot as a blueprint to make easier our deployment process, after that we can begin the migrations and proper modification to integrate our old bot.